contracts/sfc/SFC.sol

```diff
@@ -264,17 +259,20 @@ contract SFC is Initializable, Ownable, Version {
         _syncValidator(validatorID, true);
     }
 
+    /// Set redirection authorizer for approving redirections
     function setRedirectionAuthorizer(address v) external onlyOwner {
         if (redirectionAuthorizer == v) {
             revert SameRedirectionAuthorizer();
         }
         redirectionAuthorizer = v;
     }
 
+    // Announce redirection of address to another address
     function announceRedirection(address to) external {
         emit AnnouncedRedirection(msg.sender, to);
     }
 
+    // Initiate redirection request of address to another address
```

Comment:
Initiate redirection of withdrawals/claims for a compromised validator account. Needs to be accepted by validator key holder.